wiiu/vino/vino.ts

```diff
@@ -653,61 +740,61 @@ export default interface VinoAPI {
 	 * Creates a text post on Miiverse
 	 * @param body - Post body
 	 * @param topicTag - Post topic_tag
-	 * @param communityID - Uint32 community ID
+	 * @param feeling - Mii feeling
 	 * @param spoiler - True if post contains spoilers
 	 * @param searchKey1 - Posts 1st search_key
 	 * @param searchKey2 - Posts 2nd search_key
 	 * @param searchKey3 - Posts 3rd search_key
 	 * @param searchKey4 - Posts 4th search_key
 	 * @param searchKey5 - Posts 5th search_key
-	 * @returns Unknown
+	 * @returns true if request started with success
 	 */
-	olv_postText: (body: string, topicTag: string, communityID: number, spoiler: boolean, searchKey1: string, searchKey2: string, searchKey3: string, searchKey4: string, searchKey5: string) => boolean;
+	olv_postText: (body: string, topicTag: string, feeling: MiiFeeling, spoiler: boolean, searchKey1: string, searchKey2: string, searchKey3: string, searchKey4: string, searchKey5: string) => boolean;
```

Comment:
This goes for all instances of this change. Where did this come from? I distinctly remember this not being the case, that this value was the community ID